components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -803,4 +806,112 @@ public void testUpdateUserWithUsernameChange() throws Exception {
 
         assertTrue("UserName claim update is not properly handled.", hasExpectedBehaviour);
     }
+
+    @Test
+    public void testListUsersWithPost() throws Exception {
+
+        String tenantDomain = "carbon.super";
+        SearchRequest searchRequest = new SearchRequest();
+        List<Object> listOfUsers = new ArrayList<>();
+
+        Map<String, Boolean> requiredAttributes = new HashMap<>();
+        SCIMUserManager scimUserManager = spy(new SCIMUserManager(mockedUserStoreManager,
+                mockClaimMetadataManagementService, tenantDomain));
+        doReturn(listOfUsers).when(scimUserManager).listUsersWithGET(any(), any(), any(), anyString(),
+                anyString(), anyString(), anyMap());
```

Comment:
Line breaking are not in the expected format. Please recheck.
```suggestion
        doReturn(listOfUsers).when(scimUserManager)
                    .listUsersWithGET(any(), any(), any(), anyString(), anyString(), anyString(), anyMap());
```